src/frontend/komponenter/Modal/UlagredeKomponenterModal.tsx

```diff
@@ -66,6 +66,9 @@ const UlagredeKomponenterModal: FC = () => {
                     tittel={
                         'Du har ikke lagret dine siste endringer og vil miste disse om du forlater siden'
                     }
+                    umamiId={
+                        'du-har-ikke-lagret-dine-siste-endringer-og-vil-miste-disse-om-du-forlater-siden'
```

Comment:
Skulle vi ha kortet disse ned litt? Så slipper vi så lange labels i grafene i umami, og id-en blir litt løsere koplet til teksten, i fall vi endrer den i fremtiden
```suggestion
                        'ulagret-innhold-advarsel'
```